lib/tasks/sdk.rake

```diff
@@ -39,6 +39,8 @@ task 'setup_env' do
   `cp #{gem_home}/lib/config/datadog.conf #{ENV['SDK_HOME']}/embedded/dd-agent/datadog.conf`
   # This sometimes causes check setup to fail
   FileUtils.rm Dir.glob('setuptools*.zip')
+  # Download JMX Fetch, too
+  Rake::Task['setup_libs'].invoke
```

Comment:
Really? I didn't realize that! But that would be true anyway, what should we do about that?